api/types.go

```diff
@@ -35,9 +34,9 @@ type sandboxResource struct {
 
 // networkCreate is the expected body of the "create network" http request message
 type networkCreate struct {
-	Name        string                 `json:"name"`
-	NetworkType string                 `json:"network_type"`
-	Options     map[string]interface{} `json:"options"`
+	Name        string   `json:"name"`
+	NetworkType string   `json:"network_type"`
+	Labels      []string `json:"labels"`
```

Comment:
Now that the api and UX layer is in docker/docker as well, pls remember to repeat these in docker network API layer.